console/src/components/Common/Tooltip/Tooltip.tsx

```diff
@@ -11,18 +11,20 @@ export interface TooltipProps extends React.ComponentProps<'i'> {
   placement?: 'right' | 'left' | 'top' | 'bottom';
   className?: string;
   tooltipStyle?: string;
+  icon?: string;
 }
 
 const ToolTip: React.FC<TooltipProps> = ({
   message,
   placement = 'right',
   tooltipStyle = '',
   children,
+  icon = 'fa-info-circle',
 }) => (
   <OverlayTrigger placement={placement} overlay={tooltipGen(message)}>
     {children || (
       <i
-        className={`fa fa-info-circle ${styles.tooltipIcon} ${tooltipStyle}`}
+        className={`fa ${icon} ${styles.tooltipIcon} ${tooltipStyle}`}
```

Comment:
Why this change?

Reply:
I made this change because we need the tooltip to be present over a different icon (the exclamation mark) than the default icon

Reply:
The idea behind this was to be consistent and always use the same icon for tooltip.

Reply:
I've reverted this change